pkg/engine/cmd.go

```diff
@@ -202,6 +203,17 @@ func (e *Engine) newCommand(ctx context.Context, extraEnv []string, tool types.T
 		cmdArgs = append(cmdArgs, f.Name())
 	}
 
+	// This is a workaround for Windows, where the command interpreter is constructed with unix style paths
+	// It converts unix style paths to windows style paths
```

Comment:
This only converts the first arg (which is ${GPTSCRIPT_TOOL_DIR}/bin/gptscript-go-tool)`. There could be some edge cases where the rest of the args contain unix path, which is not addressed here.